package.json

```diff
@@ -34,7 +34,7 @@
     "doc": "ruby script/generate-docs",
     "postdoc": "node script/update-typedoc-link.js",
     "test": "npm run test:fast",
-    "test:glob": "cross-env TS_NODE_PROJECT=test/tsconfig.json TS_NODE_CACHE=false nyc mocha --stack-trace-limit=1000  --async-stack-traces -v -r esm -rv ts-node/register -- ",
+    "test:glob": "cross-env TS_NODE_PROJECT=test/tsconfig.json TS_NODE_CACHE=false nyc --reporter=text mocha --stack-trace-limit=1000  --async-stack-traces -v -r esm -rv ts-node/register -- ",
```

Comment:
I added the `--reporter=text` here to make it easier to find out why coverage is missing.

Reply:
I dont think it is very useful. I prefer to see the summary over this. This disable the local html report in local which is very useful to view it (instead of text).

Reply:
I found the summary not very helpful to find where the missing branch was. If you prefer to use local html report, that's fine.